README.md

```diff
@@ -1 +1,235 @@
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [HTML](#HTML)
+    - [CSS](#CSS)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+> It is my first Microverse Project, and it has linter to rectify your code mistakes.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### HTML and CSS <a name="tech-stack"></a>
+
+> HTML and CSS is a coding langauge that developers use to style a website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Github
+- Git
+
+<!--
+Example command:
+
+sh
+ gem install rails
+
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+- cd my-folder
+- git clone git@github.com:myaccount/my-project.git
+<!--
+Example commands:
+
+sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+
+--->
+
+### Install
+
+Install this project with:
+- git clone 
+
+<!--
+Example command:
+
+sh
+  cd my-project
+  gem install
+
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+sh
+  rails server
+
+--->
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Ismail"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 *Author1*
+
+- GitHub: [@githubhandle](https://github.com/ismailmunyentwari9)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+>I will try to improve this project and add some new things in the future.
+
+- [ ] *[new_feature_1]*
+- [ ] *[new_feature_2]*
+- [ ] *[new_feature_3]*
```

Comment:
- [ ] KIndly add 1-3 future features that can be added to this project. This would help people know how they can contribute to developing the project. 👍